### PR TITLE
merge branch commit

### PR DESCRIPTION
**Pull request by mengfei25**
merge branch commit

### CODE REVIEW

File: docker/Dockerfile.bm [hunk 1]
```diff
@@ -66,12 +66,11 @@ RUN curl -fsSL -v -o ~/miniconda.sh -O  https://repo.anaconda.com/miniconda/Mini
 FROM dev-base AS image
 COPY --from=conda /opt/conda /opt/conda
 RUN pip install torch torchvision torchaudio torchdata torchtext --index-url https://download.pytorch.org/whl/cpu
-ARG TORCH_BENCH_BRANCH=main
-ARG TORCH_BENCH_COMMIT=${TORCH_BENCH_BRANCH}
+ARG TORCH_BENCH_COMMIT="main"
```
Comment (chuanqi129) [inline docker/Dockerfile.bm:69]: can we remove the " to keep same style?

File: docker/Dockerfile.bm.arm [hunk 2]
```diff
@@ -63,12 +63,11 @@ RUN curl -fsSL -v -o ~/miniconda.sh -O  https://repo.anaconda.com/miniconda/Mini
 FROM dev-base AS image
 COPY --from=conda /opt/conda /opt/conda
 RUN pip install torch==2.0.1 torchvision torchaudio torchdata torchtext --no-deps --index-url https://download.pytorch.org/whl/cpu
-ARG TORCH_BENCH_BRANCH=main
-ARG TORCH_BENCH_COMMIT=${TORCH_BENCH_BRANCH}
+ARG TORCH_BENCH_COMMIT="main"
```
Comment (chuanqi129) [inline docker/Dockerfile.bm.arm:66]: Same here

File: docker/Dockerfile.dnn [hunk 3]
```diff
@@ -66,46 +66,40 @@ RUN curl -fsSL -v -k -o ~/miniconda.sh -O  https://repo.anaconda.com/miniconda/M
 
 FROM dev-base AS build
 ARG PT_REPO=https://github.com/pytorch/pytorch
-ARG PT_BRANCH=main
-ARG PT_COMMIT=${PT_BRANCH}
-ARG DNNL_BRANCH=rls-v3.3
+ARG PT_COMMIT="main"
```
Comment (chuanqi129) [inline docker/Dockerfile.dnn:69]: same here

File: scripts/modelbench/launch.sh [hunk 4]
```diff
@@ -1,62 +1,51 @@
...
+        if [[ "${var}" == "EXTRA="* ]];then
+            EXTRA="${@/*EXTRA=}"
+            break
+        else
+            echo "$var" >> tmp.env
+        fi
+        shift
+    done
+    source tmp.env && rm -rf tmp.env
+fi
```
Comment (chuanqi129) [inline scripts/modelbench/launch.sh:45]: This optimize method looks great! Just have 1 small concern about the nested source tmp.env, for example, the entrance.sh will source the itself temp.env, and then the entrance.sh call the launch.sh, and the launch.sh will also source itself temp.env again, will it has potential issue?

**Review by chuanqi129 — APPROVED** [review]
LGTM! Some minor comments. And let's try your PR branch for the aws_benchmark job firstly, and merge it after CNY holiday and there is no issue.

File: scripts/modelbench/launch.sh (hunk 4 above)
Comment (mengfei25) [reply scripts/modelbench/launch.sh:45]: No, before/after source will clean the tmp.env. they are from different env file exactly, and env context is from itself params. 
Fix an issue 'echo "" > tmp.env'

File: docker/Dockerfile.bm (hunk 1 above)
Comment (mengfei25) [reply docker/Dockerfile.bm:69]: Sure

**Comment by chuanqi129** [comment]
@WeizhuoZhang-intel @zxd1997066 please help to review this change also, in case miss the potential impact of our current jobs.

**Review by WeizhuoZhang-intel — APPROVED** [review]
LGTM

**Comment by chuanqi129** [comment]
@mengfei25 Let's resolve the code conflict and then land it

**Comment by mengfei25** [comment]
migrate to #79

**Pull request closed without merging**